src/Fixer/ConfigurationDefinitionFixerInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer;
+
+use PhpCsFixer\OptionsResolver;
+
+/**
+ * @deprecated Will be merged into FixerInterface in 3.0
+ */
+interface ConfigurationDefinitionFixerInterface extends ConfigurableFixerInterface
+{
+    /**
+     * Defines the available configuration options of the fixer.
+     *
+     * @return OptionsResolver $configurationDefinition
+     */
+    public function getConfigurationDefinition();
+
+    /**
+     * Returns whether the fixer has configuration options.
+     *
+     * @return bool
+     */
+    public function isConfigurable();
```

Comment:
Because I wanted this interface to be merged in `FixerInterface` in 3.0 so that fixers use this method instead of an interface to tell whether they are configurable.